src/MultiReport/Keyboard.h

```diff
@@ -61,7 +61,6 @@ class Keyboard_ {
 
   uint8_t getLEDs() { return HID().getLEDs(); };
 
- protected:
   HID_KeyboardReport_Data_t _keyReport;
   HID_KeyboardReport_Data_t _lastKeyReport;
```

Comment:
Tiny bit of nitpicking (which does make the change considerably larger, though): if making these public, wouldn't it make sense to drop the underscore prefix too?

@obra, what do you think?

Reply:
@algernon you are correct.

Reply:
I'll get right on it.